### DISCUSSION TITLE
Add javascript to be executed only when a dashboard is loaded

### DISCUSSION THREAD

**Discussion by BieberGelson** [apache/superset#28320] [Q&A / Help]
Hello Superset Masters!
I have created an environment in python and installed Superset via pip, after some flask configuration, it is working in Windows.
Now I want to create a javascript file that will only be executed when I load dashboards, any dashboards.
Currently I was able to run a javascript file with "tail_js_custom_extra.html", but the problem is that it runs only once, if I switch from one dashboard to other, it wont load again, having this in mind, I am getting errors because the elements I want to change were not created yet on the page when I run the javascript file.
As mentioned above, I haven't downloaded via github or run a docker instance, it is a fresh pip installation.
Can you please point me where I can include the script tag? (<script type="text/javascript" src="{{ assets_prefix }}/static/....js"></script>)

What I want to do is color an entire row of a table based on a value of a cell and then hide the entire column of this cell, I cant do it using Superset functions, and the column that I will use as a base for color is not intended to be shown for the user, in summary, I cant do it directly from Superset without JS customization.

I am using Superset 4.0

Thank you for your help :)

**Comment by rodrigomar1** [comment]
Hello @BieberGelson 

Did you can resolve this problem?
